team9/templates/collegeWebPortal/professor/index.html

```diff
@@ -5,8 +5,23 @@ <h1>Welcome, {{ user.username }}</h1>
 </div>
 <div>
 	<h2>Class Schedule</h2>
-	<ul style="list-style: none;">
-		<li>You are not currently teaching any courses.</li>
-	</ul>
+	{% if not classes %}
+		<p>You are not currently teaching any courses.</p>
+	{% else %}
+	<table>
```

Comment:
Once you are ready, can apply styling from the Bootstrap CSS library. For example
`<table class="table table-bordered table-striped">`
You can find the docs at https://getbootstrap.com/docs/4.3/getting-started/introduction/. Besides that, looks good!